packages/material-ui/src/Drawer/Drawer.test.js

```diff
@@ -148,6 +148,32 @@ describe('<Drawer />', () => {
         </Drawer>
       );
 
+      it('should start closed', () => {
+        const wrapper = mount(drawerElement);
+        assert.strictEqual(wrapper.find('Modal').props().open, false);
+        assert.strictEqual(wrapper.contains('Slide'), false);
+      });
+
+      it('should open and close', () => {
```

Comment:
Why do we need these new tests for? Should we move them to the Modal?

Reply:
Happy to revert them and not add them - I added them primarily whilst I was investigating the issues raised by fixing `Modal`, as it broke the test (see the first comment of this PR).

Reply:
Alright. I think that we can focus on adding a failing test case :). Keep it simple;